src/main/java/com/gtnewhorizon/gtnhlib/core/GTNHLibCore.java

```diff
@@ -64,4 +84,15 @@ public String getMixinConfig() {
     public List<String> getMixins(Set<String> loadedCoreMods) {
         return Mixins.getEarlyMixins(loadedCoreMods);
     }
+
+    @Override
+    public boolean registerBus(EventBus bus, LoadController controller) {
```

Comment:
lol, I had no idea this was even a thing

Reply:
Also - is this something generally needed?  Not immediately clear why it's needed here vs your gt5u example pr

Reply:
Its only purpose is to fetch the classes the transformer needs to visit before the regular mods are constructed, this is needed for the annotation to work in the @Mod annotated class along its proxies.